### DISCUSSION TITLE
The Scarlet Apocalypse QUEST BUG

### DISCUSSION THREAD

**Discussion by Seigneur-fantome** [azerothcore/azerothcore-wotlk#5989] [General]
hi all, i find bug for https://fr.wowhead.com/quest=12778/l-apocalypse-%C3%A9carlate

![image](https://user-images.githubusercontent.com/79913664/119159759-db5a9580-ba57-11eb-8688-b855a57b07e2.png)

![image](https://user-images.githubusercontent.com/79913664/119160078-2e344d00-ba58-11eb-924d-1fcb64fc05ff.png)

the liche king isn't "on" the black support but "under"....

**Comment by Kitzunu** [comment]
Make a bug report

**Comment by FrancescoBorzi** [comment]
@Seigneur-fantome please open a bug report: https://github.com/azerothcore/azerothcore-wotlk/issues/new?assignees=&labels=&template=bug_report.md&title=
